packages/uniorg-parse/src/parser.ts

```diff
@@ -1486,6 +1491,16 @@ class Parser {
     return u('strike-through', { contentsBegin, contentsEnd }, []);
   }
 
+  private parseStatisticsCookie(): StatisticsCookie | null {
+    const m = this.r.lookingAt(/\[[0-9]*(\%|\/[0-9]*)\]/);
+    if (!m) return null;
+    const begin = this.r.offset() + m.index;
+    const end = begin + m[0].length;
+    const value = this.r.substring(begin, end);
+    this.r.resetOffset(end + 1);
```

Comment:
❗ this one is introducing a small bug because it skips one character after the cookie regardless of whether it's a whitespace or not:
```
* [/]hello
# parses as cookie + "ello"
* [/]  hello
# parses as cookie + " hello" (leading whitespace)
```

Reply:
🙊 I totally missed this. Thanks for catching this one. 🙏🏿